src/api/forms/draft-form-definition-repository.js

```diff
@@ -38,6 +40,18 @@ export async function create(id, formDefinition) {
   await uploadToS3(formDefinitionFilename, formDefinitionString)
 }
 
+/**
+ * Copy the draft form to live in the Form Store
+ * @param {string} id - id
+ */
+export async function createLiveFromDraft(id) {
```

Comment:
Not saying this is right, but should it have the "createForm" prefix?

Otherwise in plain English it doesn't quite make sense:

1. Create a form
2. Create a live

```suggestion
export async function createFormLiveFromDraft(id) {
```

Reply:
If anything, it should be `createDefinition`, but given that "definition" is already in the repository name e.g. `draftForm**Definition**.createLiveFromDraft(formId)` it felt a little repetitive. I'll consider the naming in the [next PR](https://github.com/DEFRA/forms-manager/pull/119).